src/Compilers/CSharp/Portable/Symbols/Source/DataPropertySymbol.cs

```diff
@@ -0,0 +1,168 @@
+            : base(containingType,
+                binder,
+                syntax,
+                syntax.Type.GetRefKind(),
+                syntax.Identifier.ValueText,
+                syntax.Identifier.GetLocation(),
+                diagnostics)
+        {
+        }
+
+        public override SyntaxList<AttributeListSyntax> AttributeDeclarationSyntaxList
+            => CSharpSyntaxNode.AttributeLists;
+
+        protected override Location TypeLocation => CSharpSyntaxNode.Type.Location;
+
+        public new DataPropertyDeclarationSyntax CSharpSyntaxNode
+            => (DataPropertyDeclarationSyntax)base.CSharpSyntaxNode;
+
+        protected override bool HasPointerTypeSyntactically
+            => CSharpSyntaxNode.Type.IsPointerType();
+
+        protected override SyntaxTokenList GetModifierTokens(SyntaxNode syntax)
+            => ((DataPropertyDeclarationSyntax)syntax).Modifiers;
+
+        protected override ArrowExpressionClauseSyntax? GetArrowExpression(SyntaxNode syntax)
+            => null;
+
+        protected override bool HasInitializer(SyntaxNode syntax)
+            => ((DataPropertyDeclarationSyntax)syntax).Initializer is object;
+
+        protected override void GetAccessorDeclarations(
+            CSharpSyntaxNode syntax,
+            DiagnosticBag diagnostics,
+            out bool isAutoProperty,
+            out bool hasAccessorList,
+            out bool accessorsHaveImplementation,
+            out bool isInitOnly,
+            out CSharpSyntaxNode? getSyntax,
+            out CSharpSyntaxNode? setSyntax)
+        {
+            isAutoProperty = true;
+            hasAccessorList = false;
+            getSyntax = setSyntax = syntax;
+            isInitOnly = true;
+            accessorsHaveImplementation = false;
+        }
+
+        protected override void CheckForBlockAndExpressionBody(CSharpSyntaxNode syntax, DiagnosticBag diagnostics)
+        {
+            // Nothing to do here
+        }
+
+        protected override DeclarationModifiers MakeModifiers(
+            SyntaxTokenList modifiers,
+            bool isExplicitInterfaceImplementation,
+            bool isIndexer,
+            bool accessorsHaveImplementation,
+            Location location,
+            DiagnosticBag diagnostics,
+            out bool modifierErrors)
+        {
+            Debug.Assert(CSharpSyntaxNode.Modifiers == modifiers);
+            Debug.Assert(!isExplicitInterfaceImplementation);
+            Debug.Assert(!isIndexer);
+            Debug.Assert(!accessorsHaveImplementation);
+
+            bool isInterface = ContainingType.IsInterface;
+            var defaultAccess = DeclarationModifiers.Public;
+
+            var allowed = DeclarationModifiers.Public |
+                DeclarationModifiers.New |
+                DeclarationModifiers.Abstract |
+                DeclarationModifiers.Virtual;
+
+            if (!isInterface)
+            {
+                allowed |= DeclarationModifiers.Override;
```

Comment:
>DeclarationModifiers.Override [](start = 27, length = 29)

If you can override, you should be able to seal.

Reply:
> If you can override, you should be able to seal. 

LDM just approved "sealed" modifier on data properties.

---
In reply to: [447670522](https://github.com/dotnet/roslyn/pull/44883#discussion_r447670522) [](ancestors = 447670522)